docker_multiarch/Dockerfile.build.jetson

```diff
@@ -21,13 +21,7 @@ ADD https://api.github.com/repos/xianyi/OpenBLAS/git/refs/heads/master /tmp/open
 RUN git clone https://github.com/xianyi/OpenBLAS.git && \
     cd OpenBLAS && \
     make -j$(nproc) TARGET=ARMV8 && \
-    make install && \
-    ln -s /opt/OpenBLAS/lib/libopenblas.so /usr/lib/libopenblas.so && \
-    ln -s /opt/OpenBLAS/lib/libopenblas.a /usr/lib/libopenblas.a && \
-    ln -s /opt/OpenBLAS/lib/libopenblas.a /usr/lib/liblapack.a
-
-ENV LD_LIBRARY_PATH $LD_LIBRARY_PATH:/opt/OpenBLAS/lib
-ENV CPLUS_INCLUDE_PATH /opt/OpenBLAS/include
+    PREFIX=/usr make install
```

Comment:
Does make install openblas to the /usr/lib directories?

Reply:
yep, /usr/include and /usr/lib.